### ISSUE TITLE
Not spec conform dependency versioning in package.json

### ISSUE THREAD

**Issue by priv-kweihmann** [ashtuchkin/iconv-lite#241]
https://github.com/ashtuchkin/iconv-lite/blob/0e5377a9ca84923e41a81f94fefef8b36b75843d/package.json#L42

does use 
```
 "safer-buffer": ">= 2.1.2 < 3"
```

after reading https://docs.npmjs.com/files/package.json#dependencies, I think it should be
```
 "safer-buffer": ">= 2.1.2 <3.0.0"
```

At least the following tool https://pypi.org/project/semantic-version/ says currently used one is not spec conform

**Comment by ashtuchkin** [comment]
Fixed in 148b6bc82ce69a1c89643db55110e83513a262ce, thanks for reporting!

**Comment by ashtuchkin** [comment]
just published v0.6.1 with this fix.
